src/App.tsx

```diff
@@ -1,24 +1,166 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { FormEvent, useEffect, useState } from 'react';
+import { Todo } from './Components/Todo';
+import { Todo as TodoType } from './types/Todo';
+import { getTodosById, addTodo, removeTodo } from './api/todos';
+import { Notification } from './Components/Notification';
+import { Filter } from './Components/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10539;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [error, setError] = useState('');
+  const [filteredTodos, setFilteredTodos] = useState<TodoType[]>([]);
+  const [filter, setFilter] = useState('');
+  const [addValue, setAddValue] = useState('');
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+  const [tempTodo, setTempTodo] = useState<TodoType | null>(null);
+  const [deleteIds, setDeleteIds] = useState<number[]>([]);
+
+  useEffect(() => {
+    getTodosById(USER_ID)
+      .then(response => {
+        setTodos(response);
+      })
+      .catch(() => setError('Unable to Load todos'));
+  }, [deleteIds]);
+
+  useEffect(() => {
+    getTodosById(USER_ID)
+      .then(response => {
+        setFilteredTodos(response);
+      })
+      .catch(() => setError('Unable to Load todos'));
+  }, [todos, tempTodo]);
```

Comment:
do you need 2 useEffects here? 🤔

Reply:
yes, one useEffect makes app more buggy

Reply:
right, you don't want unnecessary re-rendering 😄